dev/com.ibm.ws.jdbc/src/com/ibm/ws/jdbc/DataSourceService.java

```diff
@@ -659,7 +659,7 @@ protected void init() throws Exception {
             isUCP = vendorImplClassName.startsWith("oracle.ucp");
             if (isUCP) {
                 if (!createdDefaultConnectionManager && !sentUCPConnMgrPropsIgnoredInfoMessage) {
-                    Set<String> connMgrPropsAllowed = Collections.singleton("enableSharingForDirectLookups");
+                    Set<String> connMgrPropsAllowed = new HashSet<String> (Arrays.asList("enableSharingForDirectLookups", ConnectionManagerService.ENABLE_CONTAINER_AUTH_FOR_DIRECT_LOOKUPS));
                     Tr.info(tc, "DSRA4013.ignored.connection.manager.config.used", connMgrPropsAllowed);
```

Comment:
Tr.info should accept a list. It would be simpler to do:
```
Tr.info(tc, "DSRA4013.ignored.connection.manager.config.used", Arrays.asList(ConnectionManagerService.ENABLE_CONTAINER_AUTH_FOR_DIRECT_LOOKUPS, "enableSharingForDirectLookups"));
```

Reply:
done.